lib/sign_in/widgets/form.dart

```diff
@@ -61,35 +58,29 @@ class _SignInFormState extends State<SignInForm> {
             padding: EdgeInsets.symmetric(horizontal: 32),
             child: SvgPicture.asset(
               services
-                  .get<AppConfig>()
+                  .config
                   .assetName(context, 'logo/logo_with_text.svg'),
               height: 64,
               alignment: Alignment.bottomCenter,
             ),
           ),
           SizedBox(height: 32),
-          ConstrainedBox(
-              constraints: BoxConstraints(
-                maxHeight: mediaQuery.size.height -
-                    400 -
-                    mediaQuery.padding.bottom -
-                    mediaQuery.padding.top,
-              ),
-              child: PrimaryButton(
-                  onPressed: () => browser.open(
-                      url: services.get<AppConfig>().webUrl('login'),
-                      options: InAppBrowserClassOptions()),
-                  child: Text(s.signIn_form_signIn))),
+          PrimaryButton(
+              onPressed: () => browser.open(
+                  url: services.config.webUrl('login'),
+                  options: InAppBrowserClassOptions()),
+              child: Text(s.signIn_form_signIn)),
```

Comment:
Please add trailing commas in multi-line widget creations.
```suggestion
          PrimaryButton(
            onPressed: () => browser.open(
                url: services.config.webUrl('login'),
                options: InAppBrowserClassOptions()),
            child: Text(s.signIn_form_signIn),
          ),
```